src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/values-property/values-property.component.spec.ts

```diff
@@ -33,6 +34,7 @@ describe('ValuesPropertyComponent', () => {
                 ValuesPropertyComponent
             ],
             imports: [
+                DotTextareaContentModule
```

Comment:
you have to test some more here, for example if the dot-textarea-content exist and has the right attributes

Reply:
Done